resources/lang/it/auth.php

```diff
@@ -23,6 +23,14 @@
     |
     */
 
+    'activation-error'       => 'Banned or Invalid Token Or Account Already Confirmed!',
+    'activation-success'     => 'Account confermato! Ora puoi accedere! ',
+    'allow-invite'           => 'Le registrazioni sono chiuse! Devi essere invitato per registrarti! Sei stato reindirizzato alla pagina di accesso! ',
     'failed'   => 'Credenziali non corrispondenti ai dati registrati.',
+    'lost-password'          => 'Dimenticata la Password?',
+    'lost-username'          => 'Dimenticato il tuo Username?'
```

Comment:
Missing a comma here at the end of the line.

Reply:
do I modify it now on the repository?

Reply:
Just update it on your branch you did for this PR.